lib/rspec/core/pending.rb

```diff
@@ -52,35 +52,31 @@ class PendingExampleFixedError < StandardError; end
       #       end
       #     end
       #
-      # @note `before(:example)` hooks are eval'd when you use the `pending`
-      #   method within an example. If you want to declare an example `pending`
-      #   and bypass the `before` hooks as well, you can pass `:pending => true`
-      #   to the `it` method:
+      # @example Alternatively, you can use the `pending` example method:
       #
-      #       it "does something", :pending => true do
+      #     describe "an example" do
+      #       pending "does not implement something yet" do
       #         # ...
       #       end
+      #     end
       #
-      #   or pass `:pending => "something else getting finished"` to add a
-      #   message to the summary report:
+      #   or specify metadata on an example:
       #
-      #       it "does something", :pending => "something else getting finished" do
-      #         # ...
-      #       end
+      #     it "does this", :pending => "is not yet implemented" do
+      #        # ...
+      #     end
+      #
+      #   even without an explicit pending message:
+      #
+      #     it "does something", :pending do
+      #       # ...
+      #     end
```

Comment:
I'm not sure if it's the best place for this doc.
Putting it to https://github.com/rspec/rspec-core/blob/11928e0d0fc3265899c74a035c7ec48ce59ffed3/lib/rspec/core/example_group.rb#L198 would be better, but I'm not sure how to refer from this doc to `example_group.rb`, would `@see {ExampleGroup#pending}` work? https://rubydoc.info/gems/yard/file/docs/GettingStarted.md#linking-objects

Reply:
This whole doc is about pending examples, if you'd like to change it can we do so in another PR with a diff of the rendered docs?

Reply:
A summary of what is done here:

1. Replaced a confusing "an example" docstring of a `describe` statement:

```diff
- describe "an example" do
+ describe "some behaviour" do
```

2. Removed the misleading notion of a possibility to bypass hooks for any notation of a pending example:

```diff
- #`before(:example)` hooks are eval'd when you use the `pending`
- #   method within an example. If you want to declare an example `pending`
- #   and bypass the `before` hooks as well,
```

just because hooks are run either way.

3. Wrapped the example of `pending` example group alias into `describe` to make a distinction between `pending` example alias and `pending` example-scope method:
```ruby
it "is pending with no message" do
  pending
end
```
vs
```ruby
describe "an example" do
  pending "does not implement something yet" do
    # ...
  end
end
```

I admit I've repeated the mistake with `"an example"` docstring I've fixed in 1) 😆
Let me fix this real quick.

There is no massive change here, but surely it's hard to grasp what was made just by looking at the diff.